llama_index/llms/anyscale.py

```diff
@@ -48,6 +49,7 @@ def __init__(
         self,
         model: str = "meta-llama/Llama-2-70b-chat-hf",
         temperature: float = 0.1,
+        max_tokens: int = 512,
```

Comment:
quick note, the default should probably be 256, similar to other parts of llama-index. There is a constant in `llama_index/constants.py` for this: `DEFAULT_NUM_OUTPUTS`

Reply:
Set to 256 now